recipes/SimpleElastix/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "0.9.0" %}
+
+package:
+  name: example
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/kaspermarstal/SimpleElastix.git
+  git_tag: 93b1482100c9fa8fba156b4d8f28f0388debc38e
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - swig
+  run:
+    - python
+
+test:
+  imports:
+    - SimpleITK
+
+about:
+  home: http://simpleelastix.github.io/
+  license: Apache
+  summary: User-friendly medical image registration library
+
+extra:
+  recipe-maintainers:
+    - Kasper Marstal
+    - Almar Klein
```

Comment:
You should put GitHub handles here. our automatic system will create teams of maintainer os GitHub based on those.